.pre-commit-config.yaml

```diff
@@ -13,15 +13,10 @@ repos:
     hooks:
       - id: black
 
-  - repo: https://github.com/PyCQA/flake8
-    rev: 7.0.0
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: v0.4.4
     hooks:
-      - id: flake8
-
-  - repo: https://github.com/timothycrosley/isort
-    rev: 5.13.2
-    hooks:
-      - id: isort
+      - id: ruff
```

Comment:
I think we'd benefit from having `ruff-format` here too.

Reply:
Instead of black?

Reply:
Oh, I forgot that we'd need to remove that too.

Yes, I think we should use ruff instead of black, but that can wait for another PR.